examples/step-49/doc/intro.dox

```diff
@@ -99,13 +99,242 @@ many cases it is best to combine both approaches. The file can be easily
 reloaded by pressing "reload" under the "Geometry" tab if you want to write
 it by hand and see the effects in the graphical user interface of gmsh.
 
+
+
 This tutorial contains an example <code>.geo</code> file that describes a box
-with two objects cut out in the interior. This is how
+with two circles cut out in the interior and several slits on the sides. This is how
 <code>example.geo</code> looks like in %Gmsh (displaying the boundary
 indicators as well as the mesh discussed further down below):
 
-<img src="https://www.dealii.org/images/steps/developer/step-49.gmsh_picture.png" alt="">
+<img src="https://www.dealii.org/images/steps/developer/step-49.gmsh_picture2.png" alt="">
+
+We will go through the basics of %Gmsh and show how we can obtain a mesh like this.
+
+It's also worth bringing up this video, https://www.youtube.com/watch?v=xL2LmDsDLYw&t=921s,
+(I don't know how to hyperlink here)
+which does a good job of explaining how to use gmsh to create basic shapes and meshes.
+Let us first go through the documented <code>example.geo</code> file that describes the geometry.
+
+Below is the documented <code>example.geo</code> file that creates the mesh.
+
+
+OpenCASCADE is CAD kernel that provides the geometry of model entities.  %Gmsh supports
+different kernels for geometry handling.  We keep this as the default.
```

Comment:
I'm not for or agaisnt the usage of the OpenCASCADE gmsh kernel, because it is a nice way to mesh when you have complex geometries.
However, you set-up the OpenCascade kernel and then proceed to use the more regular GMSH objects (points -> line -> Curve Loops -> surfaces). Why do you need to use the OpenCascade kernel? You could not specify any kernel and things would work just as fine (without confusing the reader) :). Generally, we only use the OpenCascade kernel when we want to insert 3D objects and do Boolean Fragments/Differences/etc. between them.